requirements.txt

```diff
@@ -1 +1,2 @@
 librosa>=0.4.3
+tensorflow>=0.12
```

Comment:
I did some searching and dozens of repos include tensorflow in their requirements but I only found one with tensorflow-gpu, which suggests having tensorflow-gpu already installed just ignores the regular tensorflow. It would be good to double-check though.